src/app.ts

```diff
@@ -3,10 +3,10 @@ import mongoose from 'mongoose';
 import bodyParser from 'body-parser';
 import morgan from 'morgan';
 import expressValidator from 'express-validator';
-import cors from 'cors';
```

Comment:
Why has CORS been removed?

Reply:
because it didn't work in combination with daves refactoring :D